rust/onnxruntime/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+authors = ["Nicolas Bigaouette <nbigaouette@gmail.com>"]
+edition = "2018"
+name = "onnxruntime"
+version = "0.0.14"
+
+description = "Wrapper around Microsoft's ONNX Runtime"
+documentation = "https://docs.rs/onnxruntime"
+homepage = "https://github.com/nbigaouette/onnxruntime-rs"
+license = "MIT OR Apache-2.0"
+readme = "../README.md"
+repository = "https://github.com/nbigaouette/onnxruntime-rs"
+
```

Comment:
@RyanUnderhill MS mandates that internal build makes use of internal crates repository.